spyder/config/manager.py

```diff
@@ -44,8 +44,6 @@ def __init__(self, parent=None, active_project_callback=None):
         Configuration manager to provide access to user/site/project config.
         """
         path = self.get_user_config_path()
-        if not osp.isdir(path):
-            os.makedirs(path)

```

Comment:
Why this change? This has nothing to do with this PR, so it should be reverted and discussed in another PR.

Reply:
Oh, yeah I just noticed that was repeated because the method called in the line before it does the same operation.

Reply:
I must have forgotten to submit the comment last time.  I noticed that the method in the previous line performs this exact operation, so it's repeated.